<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Mount</class>
 <widget class="QDialog" name="X2Mount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>339</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Setup Pulsar2</string>
  </property>
  <widget class="X2Form" name="X2Form">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>409</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>440</width>
     <height>390</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>0</y>
      <width>392</width>
      <height>280</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>415</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Advanced Settings</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>168</y>
       <width>248</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Possible hours past meridian (West) :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>136</y>
       <width>248</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Possible hours past meridian (East) :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>200</y>
       <width>256</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Hour Angle for automatic telescope flip :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_HoursEast">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>136</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <double>6.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_HoursWest">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>168</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <double>6.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_HourFlip">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <double>-4.000000000000000</double>
     </property>
     <property name="maximum">
      <double>4.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>24</y>
       <width>360</width>
       <height>104</height>
      </rect>
     </property>
     <property name="title">
      <string>Meridian Treatment in Controller:</string>
     </property>
     <widget class="QRadioButton" name="radioButton_Stop">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>24</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Tracking</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Flip">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>48</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Telescope Flip</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Continue">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>72</y>
        <width>321</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue to Track (this is always true in Pulsar2)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>288</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>288</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
